include/zephyr/drivers/video.h

```diff
@@ -449,7 +453,7 @@ static inline int video_stream_stop(const struct device *dev)
 	}
 
 	ret = api->stream_stop(dev);
-	video_flush(dev, VIDEO_EP_ANY, true);
```

Comment:
This is the only place I have found `VIDEO_EP_ANY` used.
If `video_endpoint_id` is a direction, "ANY" makes a lot of sense, but if allowing it to be an endpoint number, "ALL" seems less ambiguous (i.e. here flushing all endpoints rather than flushing endpoints for any direction).